cmd/kubeadm/app/cmd/token.go

```diff
@@ -92,6 +92,10 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		Use:   "delete",
 		Short: "Delete bootstrap tokens on the server.",
 		Run: func(tokenCmd *cobra.Command, args []string) {
+			if len(args) < 1 {
+				fmt.Println("missing subcommand; 'token delete' is missing token of form [\"^([a-z0-9]{6})$\"]")
```

Comment:
I went ahead and made it handled by the default kubeadm util error handler. Has exit code 1 now.